android/src/main/java/com/example/plugin_tozny/E3dbSerializer.java

```diff
@@ -0,0 +1,50 @@
+package com.example.plugin_tozny;
```

Comment:
prefer `TozStore` or Tozny to E3db

Reply:
ah the "example" slipped in, will refactor that out -- but I do prefer the namespace plugin_tozny even though its ugly... This is not a true Dart SDK which we may want in the future and it doesn't fit in TozStore since it will also have TozID, thoughts?

Reply:
Sounds like a good case for the generic `Tozny`